spec/swagger_helper.rb

```diff
@@ -250,6 +260,17 @@
               slug: { type: :string },
               image_url: { description: "Podcast image url", type: :string, format: :url }
             }
+          },
+          Tag: {
+            description: "Represetation of a tag",
+            type: "object",
+            properties: {
+              id: { description: "Tag id", type: :integer, format: :int32 },
+              name: { type: :string },
+              bg_color_hex: { description: "Background color (hexadecimal)", type: :string },
+              text_color_hex: { description: "Text color (hexadecimal)", type: :string }
```

Comment:
With the data populated (comment from above) you will likely run into some errors because these properties are nullable, so you can do this too:

```suggestion
              bg_color_hex: { description: "Background color (hexadecimal)", type: :string, nullable: true },
              text_color_hex: { description: "Text color (hexadecimal)", type: :string, nullable: true }
```